src/plugins_3rdparty/umuscle/src/MuscleParallel.cpp

```diff
@@ -64,16 +64,18 @@ int MuscleParallelTask::estimateMemoryUsageInMb(const MultipleSequenceAlignment
     }
     std::sort(rowsLengths.begin(), rowsLengths.end(), std::greater<int>());
 
-    qint64 usedBytes = 0;
+    quint64 usedBytes = 0;
     int availableThreads = workpool->nThreads;
     for (int i = 0; i < rowsLengths.size() && availableThreads > 0; i++) {
         for (int j = 0; j < rowsLengths.size() && availableThreads > 0; j++, availableThreads--) {
             usedBytes += qint64((rowsLengths[i] + 1025)) * (rowsLengths[j] + 1025);
         }
     }
     const int maxInt = std::numeric_limits<int>::max();
+    quint64 m_DistsUsedBytes = sizeof(float) * rowsLengths.size() * rowsLengths.size();
```

Comment:
Do not use Hungarian notation.

Reply:
Tried to save original variable name, moved it to comment.